packages/shopping/src/__tests__/acceptance/user.controller.acceptance.ts

```diff
@@ -91,7 +92,9 @@ describe('UserController', () => {
       })
       .expect(422);
 
-    const errorText = JSON.parse(res.error.text);
+    expect(res.error).to.not.eql(false);
```

Comment:
The `res.error` 's type is `false | HTTPError` after upgrading the dependencies.
Therefore we need a new assertion to make sure error exists, then use type assertion for the text check.